source/en/api/api_recommender.rst

```diff
@@ -296,7 +296,7 @@ Methods
    .. mpidl:method:: list<id_with_score> similar_row_from_id_and_rate(0: string id, 1: float rate)
 
       :param id:   row ID
-      :param rate: rate of all the rows to be returned
+      :param rate: rate of all the rows to be returned (Range ``0 <= rate <= 1``)
```

Comment:
range of `rate` is different between Japanese and English.
Japanese is correct according to the code.